omnibus/package-scripts/installer-deb/postinst

```diff
@@ -11,8 +11,13 @@ readonly STABLE_INSTALLER=${PACKAGES_DIR}/datadog-installer/stable/bin/installer
 readonly PACKAGES_LOCK_DIR=/var/run/datadog-packages
 
 bootstrap() {
-    $BOOTSTRAP_INSTALLER bootstrap --package "datadog-installer" && \
-    $STABLE_INSTALLER bootstrap
+    if [ -z "$DD_INSTALLER_OCI_BOOTSTRAP_VERSION" ]; then
```

Comment:
this is removed here https://github.com/DataDog/datadog-agent/pull/24920#pullrequestreview-2014970530

Reply:
will update once merged